src/overlays/gamestates/ovl_daytelop/z_daytelop.c

```diff
@@ -1,22 +1,256 @@
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00100000, 0x00000000, 0x00000000, 0x80000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000400, 0x00000000, 0x00000000, 0x00000000, 0x00000400, 0x00000000, 0x00000000, 0x00000000,
+    0x00100000, 0x00000000, 0x00000000, 0x80000000, 0x00000000, 0x00000000, 0x00000000, 0x00000100, 0x00000400,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000080, 0x00000000, 0x00000000,
+    0x00000000, 0x00000080, 0x00000000, 0x00000000, 0x00000000, 0x80000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000400, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000001, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000002, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000001, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000400, 0x00100000, 0x00000000, 0x00000000, 0x00000400, 0x00000000, 0x00000000,
+    0x00000001, 0x00000000,
+};
+
+u8 D_80815FF0 = 0;
+
+TexturePtr sDayLeftTextures[] = {
+    gDaytelopFirstDayLeftNESTex,
+    gDaytelopSecondDayLeftNESTex,
+    gDaytelopFinalDayLeftNESTex,
+
+    // Last element is the new day
+    gDaytelopNewDayLeftNESTex,
+};
+
+TexturePtr sDayRightTextures[] = {
+    gDaytelopFirstDayRightNESTex,
+    gDaytelopSecondDayRightNESTex,
+    gDaytelopFinalDayRightNESTex,
+
+    // Last element is the new day
+    gDaytelopNewDayRightNESTex,
+};
+
+TexturePtr sHoursLeftTextures[] = {
+    gDaytelop72HoursNESTex,
+    gDaytelop48HoursNESTex,
+    gDaytelop24HoursNESTex,
+};
+
+#ifdef NON_MATCHING
+// minor regalloc and swapped instruction
+void Daytelop_Update(DaytelopContext* this, GameState* gameState) {
+    s16 new_var;
+    u8 temp_v0_2;
+
+    this->transitionCountdown--;
+    if (this->transitionCountdown == 0) {
+        if (gSaveContext.day < 9) {
+            gSaveContext.gameMode = 0;
+        } else {
+            gSaveContext.unk_3F4A = 0xFFF6;
+            gSaveContext.day = 1;
+        }
+
+        // clang-format off
+        this->common.running = false; this->common.nextGameStateInit = Play_Init; this->common.nextGameStateSize = sizeof(GlobalContext);
+        // clang-format on
```

Comment:
Including the `running = false`?

Reply:
It's possible it is a part of the macro also. It isn't in OoT I don't think but it's reasonable to suspect it was changed in MM, since everywhere that this macro is used in OoT it is accompanied by a `running = false` also.